src/vlei-issuance.ts

```diff
@@ -771,6 +772,59 @@ export const VleiIssuance = {
     const schema = workflow_state.schemas[credInfo.schema];
     const rules = workflow_state.rules[credInfo.rules!];
     const privacy = credInfo.privacy;
+
+    // Extract the role from the credential info or attributes
+    const role =
```

Comment:
similar comment to above, this could potentially be an OOR so lets figure out how to make it generic enough.